File: SchoolStudy/Hello_2/app/src/main/java/com/example/hello_2/MainActivity.java
package com.example.hello_2;

import android.content.Intent;
import android.net.Uri;
import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void onButton1Clicked(View v) {      // 버튼이 눌렸을 때 동작하도록 함
        Toast.makeText(getApplicationContext(),"버튼이 눌렸어요.", Toast.LENGTH_LONG).show();      // 토스트 텍스트가 출력
    }

    public void onButton2Clicked(View v) {
        Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse("http://m.naver.com"));            // 네이버 모바일 창 열기
        startActivity(intent);
    }

}

File: SchoolStudy/Lesson11/appcompatLibrary/app/src/main/java/com/example/libraryappcompat/MainActivity.kt
package com.example.libraryappcompat

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.Menu
import android.view.MenuItem
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        title = "Main Activity"
    }

    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
        menuInflater.inflate(R.menu.menu_main, menu)
        return super.onCreateOptionsMenu(menu)
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        when(item.itemId){
            R.id.menu1 -> startActivity(Intent(this, MainActivity2::class.java))
            R.id.menu2 -> startActivity(Intent(this, MainActivity3::class.java))
            R.id.menu3 -> Toast.makeText(this, "세번째 메뉴", Toast.LENGTH_SHORT).show()
        }
        return super.onOptionsItemSelected(item)
    }
}